server/player/handler.go

```diff
@@ -122,6 +122,8 @@ type Handler interface {
 	// HandleSignEdit handles the player editing a sign. It is called for every keystroke while editing a sign and
 	// has both the old text passed and the text after the edit. This typically only has a change of one character.
 	HandleSignEdit(ctx *Context, pos cube.Pos, frontSide bool, oldText, newText string)
+	// HandleSleep handles the player going to sleep. ctx.Cancel() may be called to cancel the sleep.
```

Comment:
Should make this documentation clearer. Does "going to sleep" mean finishing sleeping or starting sleeping?